Gems/Atom/RHI/Metal/Code/Source/RHI/CommandListBase.cpp

```diff
@@ -43,9 +43,11 @@ namespace AZ
             m_mtlCommandBuffer = mtlCommandBuffer;
         }
         
-        void CommandListBase::Open(id <MTLCommandEncoder> subEncoder)
+        void CommandListBase::Open(id <MTLCommandEncoder> subEncoder, id <MTLCommandBuffer> mtlCommandBuffer)
         {
+            //This commandlist will be used for parallel encoding work and hence it will be encoded.
```

Comment:
Is this comment maybe a bit superfluous?

Reply:
The idea behind this comment is if people may wonder why m_isEncoded is true in this function but not in 
`void CommandListBase::Open(id <MTLCommandBuffer> mtlCommandBuffer)`
just above. This is to indicate that this function is use a group that has a lot of work and hence it will be encoded where as the function above maybe used by a group that just adds a fencing command or just adds a load/store action but not draw packets are submitted.